Yoast/YoastSEO.js
# Throw error when passive voice percentage is NaN

## Summary

This PR can be summarized in the following changelog entry:

* When passive voice percentage returns `NaN` (Not a number), the assessment throws an error.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Change line 24 in `passiveVoiceAssessment` to `percentage = NaN`.
* Open the example app and enter any sentence (Can be either passive or active).
* The assessment should show up as an error.

Fixes #1344
